docs/migration.md

```diff
@@ -126,10 +155,14 @@ public class MainApplication extends Application implements ReactApplication {
     [AEPMobileCore setLogLevel: AEPLogLevelDebug];
     [AEPMobileCore configureWithAppId:@"yourAppID"];
     [AEPMobileCore registerExtensions: @[
+        AEPMobileUserProfile.class,
+        AEPMobileIdentity.class,
         AEPMobileLifecycle.class,
         AEPMobileSignal.class,
-        AEPMobileIdentity.class,
-        AEPMobileUserProfile.class,
+        AEPMobileTarget.class,
+        AEPMobileCampaignClassic.class,
+        AEPMobilePlaces.class,
+        AEPMobileAssurance.class,
     ] completion:^{
     [AEPMobileCore lifecycleStart:@{@"contextDataKey": @"contextDataVal"}];
```

Comment:
We need to check if the app is not in background before calling lifecycleStart.
Can you also update the other places with similar logic?